app/pages/cif/project/[project]/index.tsx

```diff
@@ -1,29 +1,145 @@
 import DefaultLayout from "components/Layout/DefaultLayout";
 import { withRelay, RelayProps } from "relay-nextjs";
 import { graphql, usePreloadedQuery } from "react-relay/hooks";
-import { ProjectOverwiewQuery } from "__generated__/ProjectOverwiewQuery.graphql";
+import { ProjectOverviewQuery } from "__generated__/ProjectOverviewQuery.graphql";
 import withRelayOptions from "lib/relay/withRelayOptions";
 
-const pageQuery = graphql`
-  query ProjectOverwiewQuery($project: ID!) {
+export const pageQuery = graphql`
+  query ProjectOverviewQuery($project: ID!) {
     session {
       ...DefaultLayout_session
     }
     project(id: $project) {
       projectName
+      rfpNumber
+      totalFundingRequest
+      summary
+      operatorByOperatorId {
+        legalName
+        bcRegistryId
+        tradeName
+      }
+      fundingStreamRfpByFundingStreamRfpId {
+        year
+        fundingStreamByFundingStreamId {
+          description
+        }
+      }
+      projectStatusByProjectStatusId {
+        name
+      }
+      contactsByProjectContactProjectIdAndContactId {
+        edges {
+          node {
+            id
+            familyName
+            givenName
+          }
+        }
+      }
+      projectManagersByProjectId {
+        edges {
+          node {
+            cifUserByCifUserId {
+              firstName
+              lastName
+              id
+            }
```

Comment:
I still don't totally follow why I sometimes have to go through edges and nodes but sometimes I just go straight to the field I'm looking for